programs/cleos/main.cpp

```diff
@@ -3952,7 +3963,6 @@ int main( int argc, char** argv ) {
    trxSubcommand->add_option("--signature", extra_sig_opt_callback, localized("append a signature to the transaction; repeat this option to append multiple signatures"))->type_size(0, 1000);
    add_standard_transaction_options_plus_signing(trxSubcommand);
    trxSubcommand->add_flag("-o,--read-only", tx_read_only, localized("Specify a transaction is read-only"));
-   trxSubcommand->add_flag("-t,--return-failure-trace", tx_rtn_failure_trace, localized("Return partial traces on failed transactions, use it along with --read-only)"));
```

Comment:
This should still be an option to match `push action`

Reply:
I see. so if we want it for every single transaction, maybe to add it elsewhere? basically every command that calls `send_actions`. this includes `action`, `transaction`, `msig propose_trx`, etc.

Reply:
Or just remove it everywhere. Not sure we need a cleos option not to return a trace.

Reply:
option not to return trace is handy when locally testing something. returned trace is verbose

Reply:
Fair enough, just make it one of the "global" options then.

Reply:
done